macro-fullcalendar-api/src/main/java/org/xwiki/fullcalendar/model/MoccaCalendarEvent.java

```diff
@@ -0,0 +1,362 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.fullcalendar.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.xwiki.stability.Unstable;
+
+/**
+ * Represents the data of a mocca calendar event.
+ *
+ * @version $Id$
+ * @since 2.4.0
+ */
+@Unstable
+public class MoccaCalendarEvent
```

Comment:
Shouldn't this name be related to FullCalendar (or general), rather than MoccaCalendar? It feels strange for this app to use namings related to another app. Even if you know that it will be used by it for now, this is a public API
Maybe simply `CalendarEvent`?